src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -1027,7 +1028,10 @@ public int size()
             }
         }
 
-        logger.info("Took {} to slice way {}", time.untilNow(), identifier);
+        if (logger.isDebugEnabled() && time.untilNow().isMoreThan(Duration.ONE_MINUTE))
+        {
+            logger.info("Took {} to slice way {}", time.untilNow(), identifier);
```

Comment:
Might want to do `logger.debug` here.

Reply:
Oops, had this and it changed back to info after fixing the branch, nice catch!